tests/gui/components/update-notifier.spec.js

```diff
@@ -4,10 +4,19 @@ const m = require('mochainon');
 const angular = require('angular');
 const units = require('../../../lib/shared/units');
 const settings = require('../../../lib/gui/models/settings');
+const analytics = require('../../../lib/shared/analytics');
 require('angular-mocks');
 
 describe('Browser: UpdateNotifier', function() {
 
+  before(function() {
+    analytics.disableConsoleOutput();
+  });
+
+  after(function() {
+    analytics.enableConsoleOutput();
```

Comment:
...would it make more sense to just disable analytics altogether during the unit-tests?